syntax = "proto3";

option go_package = "get_user/pb";

service get_users {
  rpc GetData(GetDataRequest) returns (GetDataResponse) {}
}

message User {
  int32  id        = 1;
  string name      = 2;
  string family    = 3;
  int32  age       = 4;
  string sex       = 5;
  string createdAt = 6;
}

message GetDataRequest {
  int32  user_id    = 1;
  int64 auth_key   = 2;
  int32  message_id = 3;
  string redis_key  = 4;
}


message GetDataResponse {
  repeated User return_users = 1;
  int32    message_id        = 2;

}
